src/index.ts

```diff
@@ -241,7 +241,7 @@ export class Connection {
       insertId,
       size: rows.length,
       statement: sql,
-      time
+      time: timingSeconds / 1000
```

Comment:
Don't you mean `*1000`? Since the API is expecting to return milliseconds, but we're giving you seconds. So seconds -> ms is a 1000x multiple, not divisor.

Reply:
*facepalm* You're right, I'm sorry.

Reply:
But either way, up to you, I think it's valuable that we provide both timings rather than just changing this out from under it. One is client side, and other is server side. I think it'd be worth being able to show both, say, in our web console even, might be nice to show the network latency along with the query time.

Reply:
Resolved in https://github.com/planetscale/database-js/pull/77/commits/6891f1c8e350a2901b3f2a9875b43b22ebf48465